businesscentral-monitoring/branch-overrides.yaml

```diff
@@ -10,15 +10,19 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-31
+                  ref: sprint-35
           - name: jboss-eap-modules
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: sprint-33
+                  # TODO: revert to "jboss-container-images" after this PR merges: https://github.com/jboss-container-images/jboss-eap-modules/pull/84
+                # url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  url: https://github.com/luck3y/jboss-eap-modules.git
+                  # TODO: replace with GA tag when available
+                  ref: 7.2.x-openjdk11
```

Comment:
Wouldn't be better merge this changes on a 'snapshot' branch before push to master?

Reply:
@spolti Ken just said he's hoping on merging that PR later today, so hopefully I can get rid of this developer-branch dependency soon.

Reply:
were the related pr merged?